src/components/close-button/__tests__/CloseButton.stories.test.tsx

```diff
@@ -0,0 +1,9 @@
+describe('components/close-button/CloseButton', () => {
+    test.each(['components-buttons-closebutton--regular', 'components-buttons-closebutton--disabled'])(
```

Comment:
I think `components-buttons-closebutton--disabled` can be removed since there's no disabled in the `stories.tsx`.

Reply:
Then this `src/components/close-button/__tests__/__image_snapshots__/close-button-stories-test-tsx-components-close-button-close-button-looks-visually-correct-when-using-story-components-buttons-closebutton-disabled-1-snap.png` can be removed.